internal/run/consumer.go

```diff
@@ -141,19 +142,35 @@ func (c *Consumer) Consume(log *zap.Logger, repository *model.Repository) (err e
 		// set common annotations for this yaml
 		tags := cloneMap(repository.Tags)
 		annotations := cloneMap(repository.Annotations)
+		inferredTags := []string{}
+
+		if !c.EffxClient.IsFeatureDisabled("LANGUAGE_DETECTION") {
```

Comment:
We probably want to define this "LANGUAGE_DETECTION" string as a const somewhere. Maybe a `features.go` file in the same directory?

Reply:
good call I like that!